src/sensor/ping.h

```diff
@@ -36,23 +36,7 @@ class Ping : public Sensor
     /**
      * @brief debug function
      */
-    void printStatus();
-
-    /**
-     * @brief Get device source ID
-     *
-     * @return uint8_t source id
-     */
-    uint8_t srcId() { return _srcId; }
-    Q_PROPERTY(int srcId READ srcId NOTIFY srcIdUpdate)
-
-    /**
-     * @brief Return destiny ID
-     *
-     * @return uint8_t destiny id
-     */
-    uint8_t dstId() { return _dstId; }
-    Q_PROPERTY(int dstId READ dstId NOTIFY dstIdUpdate)
+    void printSensorInformation() const final;
```

Comment:
You should perhaps add `override` to be explicit. @tcanabrava

Reply:
final implies in override after c++11 if I'm not mistaken. `override final`is a bit redundant.

Reply:
> final implies in override after c++11 if I'm not mistaken.

Correct. The suggestion is **to be explicit**.

> override final is a bit redundant.

Not really, they are independent specifiers with independent meaning. You have just created a situation where the compiler can make an implicit decision based on the language specification. This is not clear to everyone.

Reply:
Done.